envconfig/config.go

```diff
@@ -177,16 +177,19 @@ func RunnersDir() (p string) {
 			slog.Error("unable to locate llm runner directory. Set OLLAMA_RUNNERS_DIR to the location of 'ollama/runners'")
 		}
 	}()
+	return GetRelativeRunnerDir()
+}
 
+func GetRelativeRunnerDir() string {
```

Comment:
```suggestion
func relativeRunnerDir() string {
```

Reply:
This is used both within the `gpu` package as well as `llm` so it needs to remain exposed.